src/css/components/_captions-settings.scss

```diff
@@ -2,9 +2,8 @@
   background-color: $primary-background-color;
   background-color: rgba($primary-background-color, 0.75);
   color: $primary-foreground-color;
-  height: 70%;
+  height: 100%;
```

Comment:
oh, I just noticed this. We want to keep this to 70% because we use the currently set captions as a preview (probably not ideal but that's how it is).